src/neo/Wallets/SQLite/UserWallet.cs

```diff
@@ -63,13 +67,28 @@ private UserWallet(string path, byte[] passwordKey, bool create)
                 if (passwordHash != null && !passwordHash.SequenceEqual(passwordKey.Sha256()))
                     throw new CryptographicException();
                 this.iv = LoadStoredData("IV");
+                this.passphrase = CreatePassphrase(passwordKey, iv);
                 this.masterKey = LoadStoredData("MasterKey").AesDecrypt(passwordKey, iv);
```

Comment:
Is it equivalent to doing twice encryption, is this necessary?Do we need AES to protect NEP2KEY again?

Reply:
Agree, We can remove it

Reply:
If you want to change password easily, you will need `MasterKey`. Otherwise, you have to decrypt all the keys, change the password, then re-encrypt all the keys.

Reply:
As @erikzhang said, we can use `Master Key` like the Bitlocker mechanism, it could make automatically the change password.

Reply:
I think that now it's better